### ISSUE TITLE
Support torch.export.export through torch.onnx.dynamo_export

### ISSUE THREAD

**Issue by thiagocrepaldi** [pytorch/pytorch#109889]
### 🐛 Describe the bug

Currently, `torch.onnx.dynamo_export` leverages `torch._dynamo.export` to obtain a `GraphModule` graph. By leveraging `torch.export`, we may have a more stable and complete `GraphModule` graph, including:

* Functionalization.
* Lowering from torch ops to aten ops (aten ops are a lot more normalized than torch ops)
* Decomposition of Aten ops to a smaller opset (core aten ops)

@SherlockNoMad  FYI

### Versions

pytorch main
```[tasklist]
### Tasks
- [ ] https://github.com/microsoft/onnxscript/issues/1077
- [ ] https://github.com/pytorch/pytorch/issues/110100
```
